### PR TITLE
chore( bridge-ui): update env readme

### PR DESCRIPTION
**Pull request by shadab-taiko**

### CODE REVIEW

**Review by jscriptcoder — APPROVED** [review]

File: packages/bridge-ui/README.md [hunk 1]
```diff
@@ -14,32 +14,31 @@ You can use the following values in the `.env` file to spin up the Bridge UI loc
 
 ```
 VITE_NODE_ENV=dev
+VITE_L1_RPC_URL=<sepolia_rpc>
+VITE_L2_RPC_URL="https://l2rpc.a2.taiko.xyz/"
 
-VITE_L1_RPC_URL="https://l1rpc.internal.taiko.xyz/"
-VITE_L2_RPC_URL="https://l2rpc.internal.taiko.xyz/"
+VITE_L1_EXPLORER_URL="https://sepolia.etherscan.io/"
+VITE_L2_EXPLORER_URL="https://l2explorer.a2.taiko.xyz/"
```
Comment (d1onys1us) [inline packages/bridge-ui/README.md:21]: i think `explorer.a2.taiko.xyz` is the better rpc

File: packages/bridge-ui/README.md [hunk 2]
```diff
@@ -14,32 +14,31 @@ You can use the following values in the `.env` file to spin up the Bridge UI loc
 
 ```
 VITE_NODE_ENV=dev
+VITE_L1_RPC_URL=<sepolia_rpc>
+VITE_L2_RPC_URL="https://l2rpc.a2.taiko.xyz/"
```
Comment (d1onys1us) [inline packages/bridge-ui/README.md:18]: same here

File: packages/bridge-ui/README.md (hunk 1 above)
Comment (jscriptcoder) [reply packages/bridge-ui/README.md]: Corrected

File: packages/bridge-ui/README.md (hunk 2 above)
Comment (jscriptcoder) [reply packages/bridge-ui/README.md]: Corrected

File: packages/bridge-ui/README.md (hunk 1 above)
Comment (shadab-taiko) [reply packages/bridge-ui/README.md]: We don't have any explorer.a2.taiko.xyz subdomain. @d1onys1us not sure what you meant?

**Review by dionysuzx — APPROVED** [review]

**Pull request merged**